requirements.txt

```diff
@@ -1,4 +1,10 @@
 -e  git+https://github.com/ckan/ckan.git@ckan-2.8.3#egg=ckan
+
+# datapusher hotfix: CKAN has an error for running with docker. 
+# Fixed in master, need to port to CKAN 2.8.3
+# https://github.com/ckan/ckan/pull/5281/files
+# -e git+https://github.com/avdata99/ckan@my_2.8.3#egg=ckan
```

Comment:
Is your CKAN fix _required_ before this PR is merged? Can you please push this to the GSA repo as a new branch `datagov-inventory-2.8`? I just added you as a collaborator https://github.com/GSA/ckan/invitations